gddragonbones.cpp

```diff
@@ -177,10 +178,16 @@ void GDDragonBones::dispatch_event(const String& _str_type, const EventObject* _
 				}
 		}
 
-		emit_signal("dragon_anim_event", String(_p_value->animationState->name.c_str()), String(_p_value->name.c_str()),
-				Variant(int_val),
-				Variant(float_val),
-				String(string_val));
+		Dictionary dict = Dictionary();
+
+		dict[String("armature")] = String(armature->getName().c_str());
```

Comment:
I was hitting a 5 argument limit on signals so I returned the data in a dict that I can make as big as I want. Having armature here can help filtering in the future?